models/public/hrnetv2/hrnetv2.md

```diff
@@ -0,0 +1,110 @@
+# hrnetv2
+
+## Use Case and High-Level Description
+HRNetV2-W48 is semantic-segmentation model based on architecture described in paper
+[High-Resolution Representations for Labeling Pixels and Regions](https://arxiv.org/abs/1904.04514).
+This is PyTorch implementation based on retaining high resolution representations throughout the model
+and pretrained on ADE20k dataset. The model input is a blob that consists of a
+single image of "1x3x320x320" in RGB order.
+For details about implementation of model, check out the [Semantic Segmentation on MIT ADE20K dataset in PyTorch](https://github.com/CSAILVision/semantic-segmentation-pytorch).
+
+## Example
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Segmentation  |
+| GFLOPs            | 81.9930        |
+| MParams           | 66.4768        |
+| Source framework  | PyTorch\*  |
+
+## Accuracy
+
+| Metric | Original model | Converted model |
+| ------ | -------------- | --------------- |
+| Pixel accuracy  | 77.69%          | 72.81%           |
+| mean IoU  | 33.02%          | 28.44%           |
+
+## Performance
+
+## Input
+
+### Original Model
+
+Image, name - `image`,  shape - `[1x3x320x320]`, format is `[BxCxHxW]`, where:
+
+- `B` - batch size
+- `H` - height
+- `W` - width
+- `C` - channel
+
+Channel order is `RGB`.
+
+### Converted Model
+
+Image, name - `input.1`,  shape - `[1x3x320x320]`, format is `[BxCxHxW]`, where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `RGB`.
```

Comment:
why not BGR?
in OMZ we prefer to have models with BGR input, because all demos written for this case